app/controllers/buyers/bulk_base_controller.rb

```diff
@@ -2,22 +2,46 @@
 
 class Buyers::BulkBaseController < FrontendController
   before_action :authorize_bulk_operations
+  before_action :initializa_errors, only: :create
+
+  def new; end
+
+  def create
+    @errors = nil
```

Comment:
if we init `@errors` in `#initializa_errors`, then probably no need to set it here? Or remove the `before_action`?